### ISSUE TITLE
Add integration tests for `genotyping` module

### ISSUE THREAD

**Issue by kabilar** [datajoint/workflow-session#16]

**Comment by CBroz1** [comment]
Proposed csv import structure following current design

```python
lab_sources_content=["source, source_name, contact_details,source_description"]
lab_sources_csv_path = pathlib.Path('./tests/user_data/lab/sources.csv')

subject_allele_content=["allele, allele_standard_name,zygosity"]
subject_allele_csv_path = pathlib.Path('./tests/user_data/subject/allele.csv')

subject_genotyping_content=["?"]
subject_genotyping_csv_path = pathlib.Path('./tests/user_data/subject/genotyping.csv')

subject_line_content=["line, line_description, target_phenotype, is_active"]
subject_line_csv_path = pathlib.Path('./tests/user_data/subject/line.csv')

subject_source_content=["allele,source_identifier,source_url,expression_data_url"]
subject_source_csv_path = pathlib.Path('./tests/user_data/subject/source.csv')

subject_strain_content=["strain, strain_standard_name, strain_desc"]
subject_strain_csv_path = pathlib.Path('./tests/user_data/subject/strain.csv')
```

**Comment by CBroz1** [comment]
Closed by #23
